Tools/gazebo_sitl_multiple_run.sh

```diff
@@ -14,8 +14,9 @@ function cleanup() {
 function spawn_model() {
 	MODEL=$1
 	N=$2 #Instance Number
+	ARR_MODEL=(`echo $MODEL | tr '_' ' '`)
 
-	if [ "$MODEL" != "iris" ] && [ "$MODEL" != "plane" ] && [ "$MODEL" != "standard_vtol" ]
+	if [[ $MODEL != "iris"* ]] && [[ $MODEL != "plane"* ]] && [[ $MODEL != "standard_vtol"* ]]
```

Comment:
I think the stars will already fail with various models such as `iris_fpv_cam`, `iris_opt_flow` and so on. Better to create a list if you think these && conditions are getting too large

Reply:
OK, I will change it using list.